### PR TITLE
Add line of site sampling along a Spinner Lidar rosette pattern

### PR DESCRIPTION
**Pull request by rcknaus**
* Added an exodus output option to data probes.  It doesn't seem like it scales well with a large number of data probes from some local tests

### CODE REVIEW

File: include/wind_energy/SyntheticLidar.h [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+};
+
+class LidarLineOfSite
+{
+public:
+  LidarLineOfSite() = default;
+  std::unique_ptr<DataProbeSpecInfo> determine_line_of_site_info(const YAML::Node& node);
+private:
+  void load(const YAML::Node& node);
+  void write_netcdf();
```
Comment (sayerhs) [inline include/wind_energy/SyntheticLidar.h]: This method is not needed?

File: include/wind_energy/SyntheticLidar.h [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  void setup_transfer();
```
Comment (sayerhs) [inline include/wind_energy/SyntheticLidar.h]: Transfers are handled by DataProbePostProcessing, so this is not required?

File: include/wind_energy/SyntheticLidar.h [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  PrismParameters innerPrism_;
+  PrismParameters outerPrism_;
+  double beamLength_{1.0};
+
+  std::array<double, 3> lidarCenter_{{0,0,0}};
+  std::array<double, 3> laserAxis_{{1,0,0}};
+  std::array<double, 3> groundNormal_{{0,0,1}};
+};
+
+class LidarLineOfSite
```
Comment (sayerhs) [inline include/wind_energy/SyntheticLidar.h]: Misspelling. Did you mean "line of sight"?

File: src/user_functions/GaussianWakeVelocityAuxFunction.C [hunk 4]
```diff
@@ -0,0 +1,76 @@
+/*------------------------------------------------------------------------*/
```
Comment (sayerhs) [inline src/user_functions/GaussianWakeVelocityAuxFunction.C]: I don't see the usage of this class anywhere. Is a commit missing, or is this code no longer needed?

File: include/wind_energy/SyntheticLidar.h (hunk 3 above)
Comment (rcknaus) [reply include/wind_energy/SyntheticLidar.h]: No.  Both spellings are valid and the existing capability is called "line of site" in Nalu currently.

**Pull request merged**